Shipr-Helm/templates/shipr-frontend-deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shipr-frontend
+  labels:
+    app: shipr-frontend
```

Comment:
Apart from some default labels, I think users should be able to provide more if necessary.

Reply:
You can add multiple labels on your convenience like environment name like this:
labels:
    app: shipr-frontend
    environment: dev

Reply:
You have not templated anything, how will users add more labels?

Reply:
Done